docs/csharp/language-reference/builtin-types/value-tuples.md

```diff
@@ -1,63 +1,60 @@
-
 You can define tuples with an arbitrary large number of elements:
 
-[!code-csharp-interactive[large tuple](snippets/shared/ValueTuples.cs#LargeTuple)]
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="LargeTuple":::
 
 ## Use cases of tuples
 
 One of the most common use cases of tuples is as a method return type. That is, instead of defining [`out` method parameters](../keywords/out-parameter-modifier.md), you can group method results in a tuple return type, as the following example shows:
 
-[!code-csharp-interactive[multiple method outputs](snippets/shared/ValueTuples.cs#MultipleReturns)]
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="MultipleReturns":::
 
 As the preceding example shows, you can work with the returned tuple instance directly or [deconstruct](#tuple-assignment-and-deconstruction) it in separate variables.
 
 You can also use tuple types instead of [anonymous types](../../fundamentals/types/anonymous-types.md); for example, in LINQ queries. For more information, see [Choosing between anonymous and tuple types](../../../standard/base-types/choosing-between-anonymous-and-tuple.md).
 
-Typically, you use tuples to group loosely related data elements. That is usually useful within private and internal utility methods. In the case of public API, consider defining a [class](../keywords/class.md) or a [structure](struct.md) type.
+Typically, you use tuples to group loosely related data elements. In public APIs, consider defining a [class](../keywords/class.md) or a [structure](struct.md) type.
 
 ## Tuple field names
 
-You can explicitly specify the names of tuple fields either in a tuple initialization expression or in the definition of a tuple type, as the following example shows:
+You explicitly specify tuple fields names in a tuple initialization expression or in the definition of a tuple type, as the following example shows:
 
 [!code-csharp-interactive[explicit field names](snippets/shared/ValueTuples.cs#ExplicitFieldNames)]
 
 If you don't specify a field name, it may be inferred from the name of the corresponding variable in a tuple initialization expression, as the following example shows:
 
-[!code-csharp-interactive[inferred field names](snippets/shared/ValueTuples.cs#InferFieldNames)]
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/shared/ValueTuples.cs" id="InferFieldNames":::
 
-That's known as tuple projection initializers. The name of a variable isn't projected onto a tuple field name in the following cases:
+The preceding example is called tuple projection initializers. The name of a variable isn't projected onto a tuple field name in the following cases:
```

Comment:
Why is this update? I don't think we can *call* an example tuple project initializers.  "That" refers to the sentence prior example, not to the example.
```suggestion
That's called tuple projection initializers. The name of a variable isn't projected onto a tuple field name in the following cases:
```

Reply:
Good point.